designcompose/src/main/java/com/android/designcompose/squoosh/SquooshTreeBuilder.kt

```diff
@@ -170,6 +176,8 @@ internal fun resolveVariantsRecursively(
     var overrideStyle: ViewStyle? = null
     var view = viewFromTree
 
+    assert(view.style.layoutStyle.hasPadding())
```

Comment:
This assert still needed? I think it can be removed.

Reply:
Sorry, hadn't cleaned it up yet.